### PR TITLE
AKLog: realized it was printing Optional() on every content item

### PR DESCRIPTION
**Pull request by ryanfrancesconi**
🤷🏽‍♂️fixed.

### CODE REVIEW

File: AudioKit/Common/Internals/AudioKitHelpers.swift [hunk 1]
```diff
@@ -184,8 +188,8 @@ extension Double {
             // exponential taper
             var adjustedMinimum: Double = 0.0
             var adjustedMaximum: Double = 0.0
-            if range.lowerBound == 0 { adjustedMinimum = 0.000_000_000_01 }
-            if range.upperBound == 0 { adjustedMaximum = 0.000_000_000_01 }
+            if range.lowerBound == 0 { adjustedMinimum = 0.00_000_000_001 }
```
Comment (aure) [inline AudioKit/Common/Internals/AudioKitHelpers.swift]: Why this change? Seems wrong to me.

Comment (ryanfrancesconi) [reply AudioKit/Common/Internals/AudioKitHelpers.swift]: oh, the swift lint did that. feel free to revert.

Comment (aure) [reply AudioKit/Common/Internals/AudioKitHelpers.swift]: I actually realized that it was correct this way, which is why I accepted the PR as is rather than ask for a change.
